### PR TITLE
Ubuntu Core 18 webinar page

### PR DESCRIPTION
**Pull request by zprood**
## Done

[Add a new engage page for Ubuntu Core 18 webinar ]

### CODE REVIEW

File: templates/engage/uc18wb.html [hunk 1]
```diff
@@ -0,0 +1,147 @@
+{% extends_with_args "engage/base_engage.html" with title="Ubuntu Core 18介绍"  %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1zAv3ouUrzQcQDw2DSefaePXcMu35BAumIMpzqmIkycs/edit{% endblock meta_copydoc %}
+{% if meta_description %}{% block meta_description %}Ubuntu 16.04下载, Ubuntu 18.04下载, Ubuntu服务器下载， Ubuntu系统下载， Ubuntu中文官网， Ubuntu OpenStack, MicroK8s, Ubuntu容器{{ meta_description }}{% endblock %}{% endif %}
+{% block content %}
+
+<section class="p-strip p-strip--dark p-takeover--directory">
```
Comment (solazio) [inline templates/engage/uc18wb.html]: you can remove `p-strip` from here

Comment (zprood) [reply templates/engage/uc18wb.html]: Done, thanks!

**Comment by solazio** [comment]
@zprood you need to update this branch before merging.

**Comment by SirSamTumless** [comment]
@solazio and @anthonydillon, @zprood Says this isn't appearing on the site even though it is merged.

**Comment by solazio** [comment]
@SirSamTumless I need to to a release. will do it now.

**Pull request merged**